packages/expo-media-library/ios/EXMediaLibrary/EXMediaLibrary.m

```diff
@@ -1026,6 +1032,25 @@ + (NSURL *)_normalizeAssetURLFromUri:(NSString *)uri
   return [NSURL URLWithString:uri];
 }
 
+- (BOOL)_checkIfAllPermissionsWereGranted:(UMPromiseRejectBlock)reject
+{
+#ifdef __IPHONE_14_0
+  NSDictionary *permissions = [_permissionsManager getPermissionUsingRequesterClass:[EXMediaLibraryCameraRollRequester class]];
```

Comment:
I'm a bit worried that we're adding a new method to permissions manager interface — this will probably fail if someone (accidentally) has older permissions interface installed 🤔

Reply:
I can use the existing method 😅  I don't like it but I can 😜
See [`b0306e8` (#10026)](https://github.com/expo/expo/pull/10026/commits/b0306e8eb0066a7d01c1ba9d41f158f814ede6f9)

Reply:
I'm fine with both solutions 👍